### PR TITLE
Add activation logs strip option.

### PR DESCRIPTION
**Pull request by lzbj**

### CODE REVIEW

**Comment by rabbah** [comment]
looks good - can you add a test.

**Comment by rabbah** [comment]
@lzbj please submit tests for enhancements and bug fixes.

**Comment by lzbj** [comment]
ok, got it, thanks for reminding, didn't notice it.

File: tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala [hunk 1]
```diff
@@ -307,6 +307,22 @@ class WskBasicUsageTests extends TestHelpers with WskTestHelpers {
       N = 5)
   }
 
+  it should "ensure timestamp and stream are stripped from log lines" in withAssetCleaner(wskprops) {
+    val name = "activationLogStripTest"
+    val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+
+    (wp, assetHelper) =>
+      assetHelper.withCleaner(wsk.action, name) { (action, _) =>
+        action.create(name, Some(TestUtils.getTestActionFilename("log.js")))
```
Comment (dubeejw) [inline tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala]: Need to commit log.js?

Comment (dubeejw) [reply tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala]: Test failed in Travis. Travis is stuck.

Comment (rabbah) [reply tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala]: thanks - fixed.

File: core/controller/src/main/scala/whisk/core/controller/Actions.scala [hunk 2]
```diff
@@ -593,7 +590,7 @@ trait WhiskActionsApi extends WhiskCollectionAPI with PostActionActivation with
    * the given original sequence.
    *
    * @param origSequence the original sequence that is updated/created which generated the checks
-   * @param the components of the a sequence to check if they reference the original sequence
+   * @param components the components of the a sequence to check if they reference the original sequence
```
Comment (dubeejw) [inline core/controller/src/main/scala/whisk/core/controller/Actions.scala]: Is this change intended?

Comment (rabbah) [reply core/controller/src/main/scala/whisk/core/controller/Actions.scala]: it's correcting a comment - want it in a separate pr?

File: core/controller/src/main/scala/whisk/core/controller/Actions.scala [hunk 3]
```diff
@@ -60,9 +60,6 @@ import whisk.core.entitlement.Privilege._
 object WhiskActionsApi {
   def requiredProperties = Map(WhiskConfig.actionSequenceMaxLimit -> null)
 
-  /** Grace period after action timeout limit to poll for result. */
-  protected[core] val blockingInvokeGrace = 5 seconds
```
Comment (rabbah) [inline core/controller/src/main/scala/whisk/core/controller/Actions.scala]: and this is dead code.

File: core/controller/src/main/scala/whisk/core/controller/Actions.scala (hunk 2 above)
Comment (dubeejw) [reply core/controller/src/main/scala/whisk/core/controller/Actions.scala]: Nah, it's fine in this one. Just checking.
